admin/views/service-manager/check-windows-form.ejs

```diff
@@ -33,11 +33,11 @@
                     <span class="error-message"><%= error.checkWindowName %></span>
                     <% } %>
                 </label>
-                <input class="form-control <%= error.checkWindowName ? 'form-control-error' : '' %>"
+                <input class="form-control <%= typeof error !== 'undefined' && error.checkWindowName ? 'form-control-error' : '' %>"
```

Comment:
With all these checks can't the error object be initialised so it isn't `undefined`?  Would simplify imo.

Reply:
@jon-shipley updated